Personal-portfolio/services.html

```diff
@@ -0,0 +1,100 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Services | Dhruvin Patel</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://emoji-css.afeld.me/emoji.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="lib/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Anaheim' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Fahkwang' rel='stylesheet'>
+    
+</head>
+<body>
+    <!-- Header starts -->
+    <header>
+        <div class="main-logo">
+            <img src="main-logo.png" />
+        </div>
+        <div>
+            <nav> 
+                
+                <a href="index.html">Home</a>
+                <a href="resume.html">Resume</a>
+                <a href="services.html">Services</a>
+                <a href="contact.html">Contact</a>
+                
+            </nav>
+        </div>
+    </header>
+    <!-- Header ends -->
+
+    <div class="service-page"  >
+        <div hidden>
+            <p>ABC</p>
+        </div>
+    </div>
+
+    <!-- Footer Starts -->
+    <footer class="footer-container">
+        <div class="footer-left">
+            <h3>Company<span>logo</span></h3>
+            <p class="footer-links">
+                <a href="index.html">Home</a>
+                ·
+                <a href="resume.html">Resume</a>
+                ·
+                <a href="services.html">Services</a>
+                ·
+                <a href="contact.html">Contact</a>
+            </p>
+            <p class="footer-company-name">Company Name © 2020</p>
+        </div>
+
+        
+
+        <div class="footer-center">
+            <div class="footer-location">
+                <i class="fa fa-map-marker"></i>
+                <p> Winnipeg, Canada</p>
+            </div>
+            <div class="footer-phone">
+                <i class="fa fa-phone"></i>
+                <p>+1(204)***4449</p>
+            </div>
+            <div class="footer-mail">
+                <i class="fa fa-envelope"></i>
+                <p><a href="mailto:pdhruvin221@gmail.com">pdhruvin221@gmail.com</a></p>
+            </div>
+        </div>
+
+        <div class="footer-right">
+            <p class="footer-company-about">
+                Curious to know more about us? Honour us by visiting our social networking plateforms.
```

Comment:
Good job @pdhruvin221! Just change these places and I think you should be good to host this as your portfolio